whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -670,13 +670,14 @@ private static void handleMultipleValuesAttributes(final RpslObject rpslObject,
                     rdapObject.setLang(attributeValuesAttribute.next().toString());
                 }
                 case COUNTRY -> {
-                    final Ip ip = (Ip)rdapObject;
-                    if (!StringUtil.isNullOrEmpty(ip.getCountry())){
-                        rdapObject.getRedactedRpslAttrs().addAll(rpslObject.findAttributes(COUNTRY));
-                        break;
+                    if (rdapObject instanceof final Ip ip) {
+                        if (!StringUtil.isNullOrEmpty(ip.getCountry())) {
```

Comment:
If its only set in IP, can you call this method from createIp in that case you do not need to type cast or use instance of? Similarly for Lang attribute call it from createEnitity and CreateIp only

Reply:
Could you create a `mapRedactions` method specifically for Ip ?

Reply:
Although then `language` is common to both IP and Organisation - maybe a separate method for that also, but call it with an Entity / RDAP object?

Reply:
Pair programming:
We use just one method with all the logic and we call that method from IPs related logic and Entity related logic.
If in the future this is applied to all the object is very easy to change to the main method